doc/source/Guidance-for-Reviewing-Code.rst

```diff
@@ -0,0 +1,108 @@
+Guidance for reviewing code
+===========================
+
+**As a Developer:**
+-------------------
+
+1.  Implement the functionality required in the issue, and push the
+    branch up to origin.
+2.  Check the code with respect to the standards in the Definition of
+    Done. Also check that new unit tests do not raise any warnings.
+3.  Create a pull request. A main reviewer must be assigned, but adding
+    multiple reviewers to the pull request is encouraged to help share
+    expertise. Please ask for help assigning reviewers if you need it.
+4.  The relevant issue should be referenced within the pull request.
+5.  Respond to the reviewers’ comments and update the pull request
+    accordingly. This will be an iterative process.
+6.  (Internal staff only:) Once the first reviewer is satisfied, move
+    the ticket forward into the “Second Review” column.
+7.  Ensure that someone is aware of their responsibility to provide a
+    second review.
+8.  Respond to the second reviewers’ comments and update the pull
+    request accordingly. This will be an iterative process.
+9.  By the end of the second review, the technical and scientific
+    changes, if required, made in the pull request should have been
+    thoroughly reviewed. If required from a scientific or technical
+    perspective, ensure that a subject matter expert is satisfied with
+    the changes.
+10. (Internal staff only:) Prior to moving the issue into the “Done”
+    column, notify the Product Owner that the issue has been completed,
+    and discuss the functionality implemented.
+
+**As a Reviewer:**
+------------------
+
+1. Issues should be assigned to a main reviewer. Adding multiple
+   reviewers to the pull request is encouraged to help share expertise.
+   Please ask for help assigning reviewers if you need it.
+
+2. All reviewers are encouraged to add comments to the pull request.
+
+3. Main reviewers should:
+
+   1. **Read the code and post in-line comments for suggested
+      alterations.**
+   2. **Ensure unit tests are run and pass.**
+   3. **Ensure command line interface acceptance tests run and pass.
+      These must be ran on the desktop using bin/improver tests as they
+      are not run by travis**
+   4. **The Acceptance Criteria defined within the associated issue has
+      been met.**
+   5. **The criteria within the Definition of Done has been satisfied.**
+   6. **Ensure their testing is documented on the issue.**
+
+4. Main reviewers should post comments to the pull request to show that
+   they have completed: i, ii, iii, iv, v, vi.
```

Comment:
The further indented bulleted list above has been relabelled with numbers rather than roman numerals, which means it no longer matches with this list of tasks to complete. Please can you use roman numerals (or letters) for the above else there is no way to distinguish between the top level numbered bullets and this inset list.

Reply:
I've fixed this. Plus it seems Sphinx requires very particular alignment.

Reply:
Apparently there's a known issue with read-the-docs inheriting the format of the parent list for sublists in some cases:
https://github.com/readthedocs/sphinx_rtd_theme/issues/484

As such this change still renders on read-the-docs as a numbered list. Using letters seems to work both locally and on read-the-docs, so I've pushed up a change to do that.

https://bens-improver-fork.readthedocs.io/en/wikitest/Guidance-for-Reviewing-Code.html

Reply:
I'm happy with your changes, and my alterations on top are tiny. As such, I'm going to merge this and close down the wiki Thanks @dmentipl.

Reply:
Thanks for looking into this and providing a solution 👍